includes/model/class-blog.php

```diff
@@ -122,7 +123,9 @@ public function get_id() {
 			return $this->get_url();
 		}
 
-		return \add_query_arg( 'author', $this->_id, \trailingslashit( \home_url() ) );
+		$id = \add_query_arg( 'author', $this->_id, \trailingslashit( \home_url() ) );
```

Comment:
Anywhere we use `\trailingslashit( \home_url() )` we could replace it with `\home_url( '/', 'https' )`